journal/modernized/02-journal.adoc

```diff
@@ -4444,8 +4484,11 @@ Robert Stebbing, Thomas Grigg, Charles Wright, John Gray, John Jolley, Samuel Hu
 Edward Keen, Richard Humphrey, Joseph Haymor, Richard Clopton, Peter Cole, Robert Grows,
 Edmund Darby, Thomas Brook, Richard Thurston, William Goodale.
 
-[.asterism]
-'''
+--
+//not sure if this should be embedded or not.^^^
+
+[.embedded-content-document.letter]
```

Comment:
Should this abstract be labeled a letter? Or even embedded at all?

Reply:
Yes, I think it should be embedded, but let's change it to `address`